### PR TITLE
Add Markdown to problem description field

### PR DESCRIPTION
**Pull request by saulotoledo**
This PR adds a markdown preview of the data inserted in the description field.

markdown_dirlididi

### CODE REVIEW

**Comment by EsdrasVP** [comment]
@CharlleDaniel Do you have something to add?

File: src/components/form/add-problem-form.js [hunk 1]
```diff
@@ -14,14 +15,24 @@ class AddProblemForm extends Component {
...
           onChange={this.props.handleProblemNameChange} />
-        <FieldGroup
-          className='form-control-field'
-          id='form-control-description'
-          componentClass='textarea'
-          type='text'
-          label='Description'
-          placeholder='Enter description'
-          onChange={this.props.handleProblemDescriptionChange} />
+        <div className='row'>
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion
        <Row>
```

File: src/components/form/add-problem-form.js [hunk 2]
```diff
@@ -14,14 +15,24 @@ class AddProblemForm extends Component {
...
+          <div className='col-sm-6'>
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion
          <Col sm={6}>
```

File: src/components/form/add-problem-form.js [hunk 3]
```diff
@@ -14,14 +15,24 @@ class AddProblemForm extends Component {
...
+            <FieldGroup
+              className='form-control-field'
+              id='form-control-description'
+              componentClass='textarea'
+              type='text'
+              label='Description'
+              placeholder='Enter description'
+              onChange={this.props.handleProblemDescriptionChange} />
+          </div>
+          <div className='col-sm-6'>
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion
            <Col sm={6}>
```

File: src/components/form/add-problem-form.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { Button, ControlLabel, FormGroup, FormControl, HelpBlock } from 'react-bootstrap';
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion
import { Button, ControlLabel, Col, Row, FormGroup, FormControl, HelpBlock } from 'react-bootstrap';
```

File: src/components/form/add-problem-form.js [hunk 5]
```diff
@@ -14,14 +15,24 @@ class AddProblemForm extends Component {
...
+              className='form-control-field'
+              id='form-control-description'
+              componentClass='textarea'
+              type='text'
+              label='Description'
+              placeholder='Enter description'
+              onChange={this.props.handleProblemDescriptionChange} />
+          </div>
+          <div className='col-sm-6'>
+            <div className='markdown-container'>
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion

```

File: src/components/form/add-problem-form.js [hunk 6]
```diff
@@ -14,14 +15,24 @@ class AddProblemForm extends Component {
...
+              <ReactMarkdown source={this.props.problemMarkdown} />
```
Comment (CharlleDaniel) [inline src/components/form/add-problem-form.js]:
```suggestion
              <ReactMarkdown className='markdown-container' source={this.props.problemMarkdown} />
```

**Comment by saulotoledo** [comment]
Thank you for the review, @CharlleDaniel ! The changes were applied.

**Pull request merged**